app/controllers/concerns/blacklight/default_component_configuration.rb

```diff
@@ -64,39 +65,26 @@ def add_show_tools_partial name, opts = {}
         end
       end
 
-      ##
       # Add a tool to be displayed for each document in the search results.
-      # @param partial [String] the name of the document partial
-      # @param opts [Hash]
-      # @option opts [Symbol,Proc] :if render this action if the method identified by the symbol or the proc evaluates to true.
-      #                             The proc will receive the action configuration and the document or documents for the action.
-      # @option opts [Symbol,Proc] :unless render this action unless the method identified by the symbol or the proc evaluates to true
-      #                             The proc will receive the action configuration and the document or documents for the action.
-      def add_results_document_tool name, opts = {}
+      # @see #add_nav_action for params
```

Comment:
The problem with duplication in docs is maintenance. It's hard enough to remember to fix the docs to match code changes when it's only _one_ place you have to change, let alone when you have to remember to find _all_ the places you have to change (and I'm not sure how you'd find them easily?). 

Blacklight (and this is not unique to BL, it's common) has not always been great historically at updating inline docs to match code changes, it hasn't been super unusual to find inline docs that are out of date and no longer accurate.